doc/api/crypto.md

```diff
@@ -42,7 +42,10 @@ added: v0.11.8
 -->
 
 SPKAC is a Certificate Signing Request mechanism originally implemented by
-Netscape and now specified formally as part of [HTML5's `keygen` element][].
+Netscape and was specified formally as part of [HTML5's `keygen` element][].
+
+Note `<keygen>` was deprecated since [HTML 5.2][] and new project should'nt use
```

Comment:
Nits:
"Note" -> "Note that" (with rewrapping within 80 characters).
"should'nt" -> "shouldn't".

Reply:
I would prefer "should not" to make it a bit more formal.